resources/META-INF/scala-plugin-common.xml

```diff
@@ -879,6 +879,10 @@
         <localInspection implementationClass="org.jetbrains.plugins.scala.codeInspection.functionExpressions.UnnecessaryPartialFunctionInspection"
                          displayName="Unnecessary Partial Function" groupPath="Scala" groupName="General" shortName="UnnecessaryPartialFunction"
                          id="UnnecessaryPartialFunction" level="WARNING" enabledByDefault="true" language="Scala"/>
+        <localInspection implementationClass="org.jetbrains.plugins.scala.codeInspection.convertNullInitializerToUnderscore.ConvertNullInitializerToUnderscore"
+                         displayName="Null initializer can be replaced by _" groupPath="Scala" groupName="General"
```

Comment:
It should go to "Syntactic Simplification" group, like [here](https://github.com/JetBrains/intellij-scala/commit/c96adc5be3cd23395b8a13418741e4bcacc0f12e).